package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "com.padoru.localization-lib",
-  "version": "0.6.0",
```

Comment:
maybe this should be a `minor` or `major` version bump since we are adding new functions? but i don't have strong feels on it.